### ISSUE TITLE
Add a "Loggin successful" or "signup successful" Alert div, when submitting form

### ISSUE THREAD

**Issue by KephaneLoemba** [diegolehyt/Project2#6]
We could add a _div_ that get's sent back to the client, for better responsiveness and "button feedback" when submitting data on login/signup forms. The div could transitioned from _opacity:1_ to _opacity: 0_ (fade out), or even _display: none;_ to make it disappear. 

Thoughts?

**Comment by diegolehyt** [comment]
I agree, we should show to the client that the action was successfully completed
